src/common/components/molecules/Modal.tsx

```diff
@@ -27,7 +27,7 @@ const Modal = ({
   showClose = true,
   overlay = true,
 }: ModalProps) => (
-  <Dialog.Root open={open} onOpenChange={setOpen} modal={focusMode}>
```

Comment:
This focus mode is causing the Modal to close itself when the focus is shifted to the mentionList component